src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
+    }
+
+    public CinemaHall(Long id, int capacity, String description) {
+        this.id = id;
+        this.capacity = capacity;
+        this.description = description;
+    }
```

Comment:
```suggestion
    public CinemaHall(Long id, int capacity, String description) {
        this.id = id;
        this(capacity,description);
    }
```

It's better to use one constructor into other